go.mod

```diff
@@ -162,3 +164,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	nhooyr.io/websocket v1.8.7 // indirect
 )
+
+replace go.viam.com/rdk => github.com/viamrobotics/rdk v0.24.0-rc0.0.20240405205559-fa9ce0d7f3bf
```

Comment:
This is needed to ensure we have at least this change from RDK: https://github.com/viamrobotics/rdk/pull/3773 so we can do rtp_passthrough